### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#1294] [Releases]
## What's Changed
* Dbus identities by @bublath in https://github.com/AsamK/signal-cli/pull/1259
* Fix: Update identities after trust by @bublath in https://github.com/AsamK/signal-cli/pull/1264
* Remove unnecessary spaces in `README.md` and add `sh` syntax highlighting by @Benjamin-Loison in https://github.com/AsamK/signal-cli/pull/1270

## New Contributors
* @Benjamin-Loison made their first contribution in https://github.com/AsamK/signal-cli/pull/1270

**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.11.11...v0.12.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by orazioedoardo** [comment]
Invalid PGP signature?

**Reply by AsamK** [reply]
I've uploaded a fixed signature

**Comment by m-ueberall** [comment]
```
[2023-08-11T11:29:20+0200] ueberall_l@desktop01:/tmp/signal-cli% gpg signal-cli-0.12.0-Linux.tar.gz.asc 2>&1 | grep "signature from"
gpg: BAD signature from "AsamK <asamk@gmx.de>" [unknown]
[2023-08-11T11:29:25+0200] ueberall_l@desktop01:/tmp/signal-cli% gpg v0.12.0.tar.gz.asc 2>&1 | grep "signature from"
gpg: Good signature from "AsamK <asamk@gmx.de>" [unknown]
```

**EDIT:** Also, you cannot currently retrieve the above key using the well-known URI--is that intentional?

```
[2023-08-11T11:40:18+0200] ueberall_l@desktop01:/tmp/signal-cli% curl https://github.com/AsamK.gpg
-----BEGIN PGP PUBLIC KEY BLOCK-----
Note: The keys with the following IDs couldn't be exported and need to be reuploaded 2BA2CD21B5B09570


=twTO
-----END PGP PUBLIC KEY BLOCK-----%                                                                                                                                                  
```

**Reply by AsamK** [reply]
I've reuploaded the gpg key to github and it seems to work again.
